api/src/main/java/org/openmrs/module/pihcore/deploy/bundle/core/EncounterTypeBundle.java

```diff
@@ -49,6 +49,8 @@ public void install() throws Exception {
         install(EncounterTypes.VACCINATION);
         install(EncounterTypes.ECHOCARDIOGRAM);
         install(EncounterTypes.MEXICO_CONSULT);
+        install(EncounterTypes.SIERRA_LEONE_OUTPATIENT_INITIAL);
+        install(EncounterTypes.SIERRA_LEONE_OUTPATIENT_FOLLOWUP);
```

Comment:
I think it makes sense to start breaking these out into implementation-specific bundles... do you have a argument as to why not do to this?

Seems weird to have Sierra Leone encounter types in Haiti or Mexico, etc...

Reply:
Yes, I think that is definitely the right path. Call it a near-future refactor?